library/ssl_tls13_generic.c

```diff
@@ -368,6 +373,13 @@ static int ssl_tls13_parse_certificate_verify( mbedtls_ssl_context *ssl,
             md_alg = MBEDTLS_MD_SHA512;
             sig_alg = MBEDTLS_PK_ECDSA;
             break;
+#if defined(MBEDTLS_X509_RSASSA_PSS_SUPPORT)
+        case MBEDTLS_TLS13_SIG_RSA_PSS_RSAE_SHA256:
```

Comment:
How about add `rsa_pss_rsae_sha384` and `rsa_pss_rsae_sha512`?
I do not think those need a new PR

Reply:
We only focus on the rsa_pss_rsae_sha256 in this task, please ignore 384 and 512 currently.